src/main/java/io/vlingo/symbio/store/state/jdbc/JDBCStorageDelegate.java

```diff
@@ -290,48 +290,28 @@ public String originatorId() {
 
   @SuppressWarnings("unchecked")
   public <R> R readAllExpressionFor(final String storeName) throws Exception {
-    final String storeNameKey = "ALL:"+storeName;
-
-    final CachedStatement<T> maybeCached = readStatements.get(storeNameKey);
-
-    if (maybeCached == null) {
-      final String select = readAllExpression(storeName);
-      final PreparedStatement preparedStatement =
-              connection.prepareStatement(
-                      select,
-                      ResultSet.TYPE_SCROLL_INSENSITIVE,
-                      ResultSet.CONCUR_READ_ONLY);
-      final CachedStatement<T> cached = new CachedStatement<>(preparedStatement, null);
-      readStatements.put(storeNameKey, cached);
-      prepareForReadAll(cached);
-      return (R) preparedStatement;
-    }
-
-    prepareForReadAll(maybeCached);
-
-    return (R) maybeCached.preparedStatement;
+    
+    final String select = readAllExpression(storeName);
+    final PreparedStatement preparedStatement =
+            connection.prepareStatement(
+                    select,
+                    ResultSet.TYPE_SCROLL_INSENSITIVE,
+                    ResultSet.CONCUR_READ_ONLY);
+    preparedStatement.clearParameters();
```

Comment:
There is no need to `clearParameters()` when the statement is new. So given your fix this is unnecessary.

Reply:
the changes has been made